ChocolateDelightsDB/3 reports.sql

```diff
@@ -0,0 +1,59 @@
+use ChocolateDelightsDB
+go
+
+--1. The amount we sell of each product (in order of most popular to least popular)
+select o.ItemDesc, QuantitySold = sum(o.Quantity)
+from Orders o
+group by o.ItemDesc
+order by QuantitySold desc
+--2. Product details (item number, description, cost of production, wholesale price)
+select distinct o.ItemNum, o.ItemDesc, o.ProductionCost, o.WholesalePrice
+from orders o
+--3. Which products brought in the most money
+select top(1) o.ItemDesc, Profit = sum((o.WholesalePrice - o.ProductionCost) * o.Quantity)
+from orders o
+group by o.ItemDesc
+order by Profit desc
+--4. Which product brought in the least money
+select top(1) o.ItemDesc, Profit = sum((o.WholesalePrice - o.ProductionCost) * o.Quantity)
+from orders o
+group by o.ItemDesc
+order by Profit 
+--5. The peak month for each product
+select o.ItemDesc, o.MonthPurchased, Profit = sum((o.WholesalePrice - o.ProductionCost) * o.Quantity)
+from orders o
```

Comment:
Can you show me only the peak month? I don't need the profit, just the month

Reply:
I'm a little stuck on this one, I'll let you know once it's fixed

Reply:
I want to know how many were sold each month. That's how we determine peak month. You can just give it to me grouped by product.

Reply:
got it, thanks

Reply:
Just tried it out. It's amazing!